polaris.shopify.com/src/components/TOC/TOC.tsx

```diff
@@ -102,25 +103,39 @@ function TOC({items}: Props) {
 
   useEffect(() => detectCurrentHeading(), [items]);
 
-  const Link = ({toId, linkText}: {toId: string; linkText: string}) => (
-    <a
-      href={`#${toId}`}
-      data-is-current={toId === idOfCurrentHeading}
-      onClick={(evt) => {
-        scrollIntoView(toId);
-        evt.preventDefault();
-      }}
-    >
-      {linkText}
-    </a>
-  );
+  const Link = ({toId, linkText}: {toId: string; linkText: string}) => {
+    const activeHeading = toId === idOfCurrentHeading;
+    return (
+      <Text
+        as="p"
+        variant="bodyMd"
+        fontWeight={activeHeading ? 'semibold' : 'regular'}
+      >
+        <a
+          href={`#${toId}`}
+          data-is-current={activeHeading}
+          onClick={(evt) => {
+            scrollIntoView(toId);
+            evt.preventDefault();
+          }}
+        >
+          {linkText}
+        </a>
+      </Text>
+    );
+  };
 
   return (
     <div className={className(styles.TOC, isNested && styles.isNested)}>
       <ul>
+        <Box paddingInlineStart="2" paddingInlineEnd="2" paddingBlockEnd="2">
```

Comment:
Suggestion: We already have a custom `Box` component in the website that is different from our polaris-react box component. I'm not particularly attached to one component over the other, but I think we should establish which one of the two `Box`es we want to use and stick with it. Since we're using other polaris-react layout components in the site now, maybe making the polaris-react `Box` the one true `Box` makes the most sense, although migrating to polaris-react `Box` throughout the rest of the site is not in scope for this work